go.mod

```diff
@@ -3,25 +3,25 @@ module decred.org/dcrdex
 go 1.15
 
 require (
-	decred.org/dcrwallet/v2 v2.0.0-20210415173800-2c9ede7f8daa
+	decred.org/dcrwallet/v2 v2.0.0-20210714172147-8815838443cd
 	github.com/btcsuite/btcd v0.20.1-beta.0.20200615134404-e4f59022a387
 	github.com/btcsuite/btcutil v1.0.2
 	github.com/davecgh/go-spew v1.1.1
-	github.com/decred/dcrd/blockchain/stake/v4 v4.0.0-20210525214639-70483c835b7f
-	github.com/decred/dcrd/certgen v1.1.2-0.20210525214639-70483c835b7f
-	github.com/decred/dcrd/chaincfg/chainhash v1.0.3-0.20210525214639-70483c835b7f
-	github.com/decred/dcrd/chaincfg/v3 v3.0.1-0.20210525214639-70483c835b7f
-	github.com/decred/dcrd/crypto/blake256 v1.0.1-0.20210525214639-70483c835b7f
-	github.com/decred/dcrd/dcrec v1.0.1-0.20210525214639-70483c835b7f
-	github.com/decred/dcrd/dcrec/edwards/v2 v2.0.2-0.20210525214639-70483c835b7f
+	github.com/decred/dcrd/blockchain/stake/v4 v4.0.0-20210715032435-c9521b468f95
+	github.com/decred/dcrd/certgen v1.1.2-0.20210715032435-c9521b468f95
+	github.com/decred/dcrd/chaincfg/chainhash v1.0.3-0.20210715032435-c9521b468f95
+	github.com/decred/dcrd/chaincfg/v3 v3.0.1-0.20210715032435-c9521b468f95
+	github.com/decred/dcrd/crypto/blake256 v1.0.1-0.20210715032435-c9521b468f95
+	github.com/decred/dcrd/dcrec v1.0.1-0.20210715032435-c9521b468f95
+	github.com/decred/dcrd/dcrec/edwards/v2 v2.0.2-0.20210715032435-c9521b468f95
 	github.com/decred/dcrd/dcrec/secp256k1/v3 v3.0.0
-	github.com/decred/dcrd/dcrjson/v3 v3.1.1-0.20210525214639-70483c835b7f
-	github.com/decred/dcrd/dcrutil/v4 v4.0.0-20210525214639-70483c835b7f
-	github.com/decred/dcrd/hdkeychain/v3 v3.0.1-0.20210525214639-70483c835b7f
-	github.com/decred/dcrd/rpc/jsonrpc/types/v3 v3.0.0-20210525214639-70483c835b7f
-	github.com/decred/dcrd/rpcclient/v7 v7.0.0-20210525214639-70483c835b7f
-	github.com/decred/dcrd/txscript/v4 v4.0.0-20210525214639-70483c835b7f
-	github.com/decred/dcrd/wire v1.4.1-0.20210525214639-70483c835b7f
+	github.com/decred/dcrd/dcrjson/v3 v3.1.1-0.20210715032435-c9521b468f95
+	github.com/decred/dcrd/dcrutil/v4 v4.0.0-20210715032435-c9521b468f95
+	github.com/decred/dcrd/hdkeychain/v3 v3.0.1-0.20210715032435-c9521b468f95
+	github.com/decred/dcrd/rpc/jsonrpc/types/v3 v3.0.0-20210715032435-c9521b468f95
+	github.com/decred/dcrd/rpcclient/v7 v7.0.0-20210715032435-c9521b468f95
```

Comment:
@vctt94 This should fix the hang at `WaitForShutdown`. https://github.com/decred/dcrd/commit/c9521b468f957479cd0802b1f5c250e9ecc1b947
I'll test this to be sure, but just FYI.

Reply:
Confirmed the fix works.  Now when using dcrd's RPC port instead of dcrwallet's I'm getting "dcrwallet.Version response missing 'dcrwalletjsonrpcapi'" as expected instead of a hang:

![image](https://user-images.githubusercontent.com/9373513/125826318-6a5d54be-fffa-4bc8-b8a7-c986b206bb13.png)